src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -204,7 +204,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             _optionCompareText = optionCompareText
             _embedVbCoreRuntime = embedVbCoreRuntime
             _suppressEmbeddedDeclarations = suppressEmbeddedDeclarations
-            _parseOptions = parseOptions
+            _parseOptions = If(parseOptions, VisualBasicParseOptions.Default) ' Required to correctly check for the presence of feature flags.
```

Comment:
@gafter  This is required if PR #12277 isn't approved, if it is *I think this can be reverted*.